src/index.js

```diff
@@ -14,8 +14,8 @@ import {
 } from 'react-router-dom';
 
 import reportWebVitals from 'reportWebVitals';
-import solrReducer from "solr-reducer";
-import { createStore } from "redux";
+import { store } from 'redux/store';
+import { Provider } from 'react-redux';
```

Comment:
We need the `Provider` component from `react-redux` to share the store with all components. Then, we can import the store in the react class component to get the store state or use `useStore` hook in the functional components.